wgpu-hal/src/metal/adapter.rs

```diff
@@ -319,6 +319,16 @@ impl crate::Adapter<super::Api> for super::Adapter {
             usage: crate::TextureUses::COLOR_TARGET | crate::TextureUses::COPY_DST, //TODO: expose more
         })
     }
+
+    unsafe fn correlate_presentation_timestamp(
+        &self,
+        user_tiemstamp_function: &mut dyn FnMut(),
+    ) -> wgt::PresentationTimestamp {
+        user_tiemstamp_function();
```

Comment:
Ahh, the glories of autocompleiton!

```suggestion
        user_timestamp_function: &mut dyn FnMut(),
    ) -> wgt::PresentationTimestamp {
        user_timestamp_function();
```

Reply:
Heh